apisix/plugins/ua-restriction.lua

```diff
@@ -97,6 +95,10 @@ function _M.check_schema(conf)
         return false, err
     end
 
+    if conf.allowlist and conf.denylist then
+        return false, "allowlist and denylist can't be enabled at the same time."
+    end
```

Comment:
I tried using `oneOf` field of jsonschema to validate configuration, but it's seems can't meet us requirement(Perhaps there is a problem with my usage). So, use code to check it directly.

Reply:
oneOf = {
        {required = {"allowlist"}},
        {required = {"denylist"}},
    },

Reply:
Both `allowlist` and `denylist` can be disabled simultaneously, so I added an additional schema. Thanks.